content/en/docs/tutorials/cluster-management/kubelet-standalone.md

```diff
@@ -0,0 +1,526 @@
+---
+title: Running Kubelet in Standalone Mode
+content_type: tutorial
+weight: 10
+---
+
+<!-- overview -->
+
+This tutorial shows you how to run a standalone Kubelet instance.
+
+You may have have different motivations for running a standalone kubelet.
+This tutorial is aimed at introducing you to Kubernetes, even if you don't have
+much experience with it. You can follow this tutorial and learn about node setup,
+basic (static) Pods, and how Kubernetes manages containers.
+
+Once you have followed this tutorial, you could try using a cluster that has a
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}} to manage pods
+and nodes, and other types of objects. For example,
+[Hello, minikube](/docs/tutorials/hello-minikube/).
```

Comment:
This sounds like a precondition of running a cluster, but it is not.

Reply:
Does it sound like a precondition? Can you rephrase it so it doesn't?

Reply:
How about just remove this paragraph?

Reply:
@sftim, do you agree to remove this paragraph?